Shared/AddNewLectureForm.razor

```diff
@@ -64,11 +65,20 @@
 
         if (createLectureModel.When < Session.When)
         {
-            errors.Add(nameof(createLectureModel.When), new (){
+            errors.Add(nameof(createLectureModel.When), new()
+            {
                 $"Session begins {Session.When:dd-MM-yyyy HH:mm}. The lecture cannot begin before then."
             });
         }
 
+        if (createLectureModel.SpeakerId == Session.Chair?.Id)
+        {
+            errors.Add("Chair", new()
```

Comment:
Pierwszy argument powinien wskazywać na pole w formularzu, które powoduje błąd walidacji. W tym przypadku jest to `nameof(createLectureModel.SpeakerId`. Obecnie błąd wygląda tak:

![image](https://user-images.githubusercontent.com/20441980/149152040-0d99e911-63ec-4602-969f-4c1b44f37935.png)

a z `SpeakerId` wygląda tak:

![image](https://user-images.githubusercontent.com/20441980/149152324-09397772-63e9-4096-96bd-ec5c6643424f.png)

(pole z wyborem speakera jest zaznaczone jako błąd, na czerwono)